### PR TITLE
Convert Hobbes to Selenium Tests

### LINKED ISSUE #1389: Convert Hobbes Tests to JUnit5/Selenium
## Feature Request

**Is your feature request related to a problem? Please describe.**
Existing Test Suit is using hobbes framework, it should move to selenium or junit framework

**Describe the solution you'd like**
Test cases should be on Selenium Framework

**Are there alternatives?**
NA

**Documentation**
Once Test Cases has been adapted to Selenium base framework, a README should explain how to run them

### PR DESCRIPTION
**Pull request by msagolj**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #1389
| Patch: Bug Fix?          | no
| Minor: New Feature?      | no
| Major: Breaking Change?  | no
| Tests Added + Pass?      | Yes
| Documentation Provided   | Yes (code comments + README.md)
| Any Dependency Changes?  |
| License                  | Apache License, Version 2.0

Our existing UI test cases are based on the now deprecated Hobbes  Test Framework. This PR start the replacement by providing a new framework based purely on open source (Junit 5 , Selenium , Selenide, AEM testing clients, aem-selenium-it-base). The PR also contains the first ~170 test cases converted from hobbes and we will continue to replace the rest in the next weeks.

### CODE REVIEW

**Comment by msagolj** [comment]
wrong destination branch should be development not master

File: testing/it/e2e-selenium/pom.xml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        <granite.it.author.url>http://localhost:4502</granite.it.author.url>
+        <granite.it.publish.url>http://localhost:4503</granite.it.publish.url>
+    </properties>
+
+    <profiles>
+        <!-- Run every available integration test. -->
+        <profile>
+            <id>test-all</id>
+
+            <!--<activation>-->
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/pom.xml:157]: Please remove these commented lines.

File: testing/it/e2e-selenium/pom.xml [hunk 2]
```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2018 Adobe
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/pom.xml:3]: 2021

File: testing/it/e2e-selenium/src/assembly/it-jar-with-dependencies.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
+<!--
+    Copyright 2020 Adobe
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/src/assembly/it-jar-with-dependencies.xml:5]: 2021

File: testing/it/e2e-selenium/src/assembly/it-module.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+<!--
+    Copyright 2020 Adobe
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/src/assembly/it-module.xml:5]: 2021

File: testing/it/e2e-selenium/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+# Running released test using generated jar with dependencies
+Make sure to download latest junit-platform-console-standalone-<version>.jar
+```
+java -jar junit-platform-console-standalone-<version>.jar -cp 'target/core.wcm.components.it.e2e-selenium-<version>-it-jar-with-dependencies.jar' --select-package "com.adobe.cq.wcm.core.components.it.seljup" --include-classname "^.*IT?$"
+```
+
+For more details on Selenium Jupiter parameters, check [selenium-jupiter user guide](https://bonigarcia.github.io/selenium-jupiter/)
+
+
+(1) https://github.com/bonigarcia/selenium-jupiter/issues/76
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/README.md:73]: Is this referenced somewhere?

File: testing/it/e2e-selenium/src/test/resources/coverage.config [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Adobe Systems Incorporated
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/src/test/resources/coverage.config:1]: 2021. However, is this even used/needed?

File: testing/it/e2e-selenium/src/test/resources/junit-platform.properties [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Adobe Systems Incorporated
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/src/test/resources/junit-platform.properties:1]: 2021

File: testing/it/e2e-selenium/src/test/resources/logback-test.xml [hunk 8]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
```
Comment (vladbailescu) [inline testing/it/e2e-selenium/src/test/resources/logback-test.xml:3]: 2021

**Review by vladbailescu — APPROVED** [review]

**Pull request merged**